community-operators/ibmcloud-iam-operator/0.1.0/ibmcloud-iam-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,506 @@
+  maturity: alpha
+  version: 0.1.0
+  replaces: ''
```

Comment:
Since this is first version, can you please remove line with `replaces` and use it later when upgrading to newer version? Thanks

Reply:
@mvalahtv that was generated by Operator Hub's bundle/package generator. But I've removed that line and submitted another commit. Please let me know.